src/v2/lib/entity-handlers.js

```diff
@@ -85,7 +90,20 @@ const getEntity = async (request, key) => {
   // e.g. contactsService.getContact(id)
   const entity = await services[key][getFn](id)
 
-  return entity || Boom.notFound(`No ${lowerCase(key)} found for ${id}`)
+  return entity || Boom.notFound(`No ${splitUppercase(key)} found for ${id}`)
+}
+
+/**
+ * The regex below is split into two parts
+ * The first part /([A-Z])/g is matching any uppercase character in the whole string
+ * (the /g means global so it matches every character not just the first one)
+ * The next part ' $1' is then inserting a space after each matched character
+ * This regex is therefore splitting the uppercase characters into its own words
+ */
+
```

Comment:
Tiny formatting tweak: remove the blank line so the comment is "joined" to the function it relates to.

```suggestion
```